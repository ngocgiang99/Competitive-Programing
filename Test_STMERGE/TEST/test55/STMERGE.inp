2
4 9
102869765 471246008 517981951 419319671 676338830 687347164 892095159 748741889 883293765 
701702402 139176007 865609088 138486928 280883232 133676598 876778249 741886484 39212905 
106858824 373177328 285495221 516284755 755759234 761295050 568942745 366886547 425812777 
719431789 487755300 940207133 554260674 919624685 463441405 895023400 907761427 721767583 
4 6
959471135 490551321 696618087 639382481 640212013 535878704 
174522755 463675161 699862465 274986231 402805248 816956214 
869330336 635057349 888451778 186817691 338374863 565427759 
999486703 810911742 121119379 711284233 989596140 155666516 
