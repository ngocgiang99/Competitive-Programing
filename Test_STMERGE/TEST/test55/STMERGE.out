142082670
535878704
